src/Home.jsx

```diff
@@ -59,29 +61,38 @@ function Home({ enterWorld }) {
               <br />
               3. Click the Next button to continue.
             </p>
-            <div className='flex mt-80 lg:mt-20 items-center'>
+
+
+            <div className='flex mt-80 lg:mt-20 items-center' style={{display:'grid', alignItems:'center'}}>
               <div
                 className='border border-black border-1 rounded-xl h-12 flex  overflow-hidden py-2 px-6 bg-white '
                 style={{
-                  borderRight: 'none',
-                  borderRadius: '0.5rem 0 0 0.5rem',
+                  borderRadius: '0.5rem',
+                  borderRightWidth: 'medium',
+                  borderBottomWidth: 'medium',
+                  background: 'aliceblue'
                 }}
               >
                 <Input onChange={setPlayerName} onSubmit={() => {}} value={playerName} />
               </div>
-              <button
-                className='text-white rounded-xl px-6 py-2  h-12  bg-gradient-to-r from-cyan-500 to-blue-500'
-                style={{
-                  borderLeft: 'none',
-                  borderRadius: '0 0.5rem 0.5rem 0',
-                }}
+
+              <button 
+              style={{
+                  //borderLeft: 'none',
+                  marginTop: '10px',
+                  width: '150px',
+                  marginLeft: '60px',
+                }} 
+                className="relative inline-block px-5 py-3 rounded-full text-white uppercase text-sm tracking-wider transition-all duration-300 overflow-hidden group"
                 onClick={() => {
                   setStoreValue('player_name', playerName)
                   enterWorld()
                 }}
-              >
-                Next
-              </button>
+                >
+                <span className="relative z-10">Next</span>
+                <span className="absolute inset-0 bg-[#1a158d] rounded-full -z-10"></span>
+                <span className="absolute inset-0 w-0 bg-[#160edb] transition-all duration-300 group-hover:w-full rounded-full -z-1"></span>
+                </button>
             </div>
             <div className='text-center text-sm w-full mt-4 cursor-pointer text-blue-700 underline' onClick={randomNameGenerator}>Random name generator</div>
```

Comment:
@princyballabh , the issue clearly mention to redesign this button i.e Random Name Generator but you have redesign the Next button please clearly read the issue.